### ISSUE TITLE
How to disable Polly flags

### ISSUE THREAD

**Issue by blockiv** [vaeth/portage-bashrc-mv#32]
In my make.conf, I've set my *FLAGS to include "-mllvm -polly", which is causing a package to break.

I'm trying to add "media-libs/openal +mllvm +polly" to  /etc/portage/package.cflags/polly, but the flags remain active. What am I doing wrong here?

**Comment by vaeth** [comment]
As you described it, it should work. When you emerge openal, it should show you some information: Do you see that the corresponding line is parsed (possible reasons if not: file/directory permissions), and if yes, do see the flags still in the lines printed at the very beginning of the emerge? (If the latter is the case then the flags might have been added later by the build system for some reason.)

**Comment by blockiv** [comment]
> As you described it, it should work. When you emerge openal, it should show you some information: Do you see that the corresponding line is parsed (possible reasons if not: file/directory permissions), and if yes, do see the flags still in the lines printed at the very beginning of the emerge? (If the latter is not the case then the flags might have been added later by the build system for some reason.)
> 

Yep, the strange part is that I do see the lines where it's properly sourcing "/etc/portage/package.cflags/polly". 

However, in the following lines, the flags persist. Initially I had all the Polly related flags defined in a variable, but even when adding them directly to CFLAGS in [my make.conf](https://bpa.st/3QQYK), they persist.

**Comment by elsandosgrande** [comment]
@blockiv Would you be willing to share the first ten or so lines of the emerge output when emerging OpenAL? Have you tried seeing if `media-libs/openal *FLAGS-="-mllvm -polly"` does the trick?
